apps/trackers/save.py

```diff
@@ -30,13 +28,3 @@ def __new__(cls, tracker, bz_api_key=None):
 
         # we should never get here
         raise BTSException("Unknown BTS")
-
-    @staticmethod
-    def assert_context(tracker):
```

Comment:
I would just merge this commit into commit a3cfe4f3081ec35b1ebc04d916bfc80b373653a2 and rename said commit to something like "Convert tracker context assertion to model validation"